tests/test_xbmc.py

```diff
@@ -28,3 +30,20 @@ def test_keyboard_env_var(self):
         kb.doModal()
         self.assertTrue(kb.isConfirmed())
         self.assertEqual(text, kb.getText())
+
+    def test_executebuiltin(self):
+        # Fetch a list of installed addons
+        cmd = dict(
+            jsonrpc='2.0',
+            method='Addons.GetAddons',
+            params={'installed': True, 'enabled': True, 'type': 'xbmc.python.pluginsource'},
+            id=1
+        )
+        result = json.loads(xbmc.executeJSONRPC(json.dumps(cmd)))
+        self.assertEqual(result.get('jsonrpc'), '2.0')
+        self.assertIsInstance(result.get('result'), dict)
+        self.assertIsInstance(result.get('result').get('addons'), list)
+
+        # Loop over the installed add-ons and execute them
+        for addon in result.get('result').get('addons'):
+            xbmc.executebuiltin('RunPlugin(plugin://%s/?abc=123)' % addon.get('addonid'))
```

Comment:
I am wondering whether this actually tests the functionality of the `xbmc.executebuiltin`. I just does the call and nothing else.

Reply:
Indeed. This function doesn't return anything in Kodi, so I can't test for this. In my tests, I write a file to a tempdir and check if it exists. I have added this do the tests of sake so it will fail in case an exception is thrown.

Reply:
Ok